index.js

```diff
@@ -141,7 +141,11 @@ function renderTag(elem, opts) {
   ) {
     tag += '/>';
   } else {
-    tag += '>';
+    if (singleTag[elem.name] && opts.recognizeSelfClosing) {
```

Comment:
`opts.recognizeSelfClosing` and `opts.xmlModel` cannot be together, hence the first `if` will lead to the `else` block.

Am I missing anything?